index.html

```diff
@@ -1711,15 +1686,109 @@ <h2>Public Keys</h2>
+implemented, including any necessary cryptographic operations.
+      </p>
 
       <p>
-Authentication is a mechanism by which an entity can prove it is the <a>DID
-subject</a>. The <em>verifier</em> of an authentication attempt can check if the
-authenticating party is presenting a valid proof of authentication, that is,
-that they are who they say they are. Note that a successful authentication on
-its own may or may not confer authority; that is up to the verifying
-application.
+There are at least two suggested methods for implementing Authorization and
+Delegation, which may be layered:
+      </p>
+
+      <ul>
+        <li>
+A <a>verifiable data registry</a> could implement a coarse-grained
+<code>controller</code> pattern by enabling <a>DID documents</a> to express
+the <a>DID</a> of another <a>DID controller</a> that controls it, or
+additionally,
+        </li>
+
+        <li>
+A <a>verifiable data registry</a> could implement a capabilities-based approach enabling
+further fine-grained control of authorization and delegation.
+        </li>
+      </ul>
+
+      <pre class="example nohighlight"
+        title="DID document with a controller property">
+{
+  "@context": "https://www.w3.org/ns/did/v1",
+  "id": "did:example:123456789abcdefghi",
+  "controller": "did:example:bcehfew7h32f32h7af3",
+  "service": [{
+    <span class="comment">// used to retrieve Verifiable Credentials associated with the DID</span>
+    "type": "VerifiableCredentialService",
+    "serviceEndpoint": "https://example.com/vc/"
+  }]
+}
+      </pre>
+    </section>
+
+    <section>
+      <section>
+        <h2>Verification relationships</h2>
+
+        <p>
+A <a>verification relationship</a> expresses the relationship between the
+<a>DID subject</a> and a <a>verification method</a>.
+        </p>
+        <p>
+A <a>DID document</a> MAY include a property expressing a specific
+<a>verification relationship</a>. In order to maximize global interoperability,
+the property SHOULD be registered in [[DID-SPEC-REGISTRIES]].
+        </p>
+        <p>
+A <a>DID controller</a> MUST be explicit about the <a>verification
+relationship</a> between the DID subject and the <a>verification method</a>.
+<a>Verification methods</a> that are not associated with a particular
+<a>verification relationship</a> MUST NOT be used for that <a>verification
+relationship</a>. See the sections below for more
+detailed examples of a <a>verification relationship</a>.
+        </p>
+
+    </section>
+
+      <section>
+      <h3>Authentication</h3>
+
+      <p>
+Authentication is a <a>verification relationship</a> which an entity can use to prove it is the <a>DID subject</a>. The <em>verifier</em> of an authentication attempt can check if the authenticating party is presenting a valid proof of authentication, that is, that they are who they say they are. Note that a successful authentication on its own may or may not confer authority; that is up to the verifying application.
```

Comment:
```suggestion
Authentication is a <a>verification relationship</a> which an entity can use to prove it is, or effectively is, the <a>DID subject</a>. The <em>verifier</em> of an authentication attempt can check if the authenticating party is presenting a valid proof of authentication, that is, that they are who they say they are. Note that a successful authentication on its own might or might not confer authority; that is up to the verifying application.
```

Reply:
You can authenticate on behalf of the DID subject and not be the DID subject... attempting to capture that here. Not entirely happy w/ the text I'm suggesting, improvements welcome.

Reply:
Reminder to break at 78 characters to make things easier on everyone else -- editors, commenters, etc.

Reply:
Thanks for reminder, cleaning this up in latest PR and have it configured locally now as well.